provider/kubernetes/kubernetes.go

```diff
@@ -367,23 +372,35 @@ func (p *Provider) loadConfig(templateObjects types.Configuration) *types.Config
 	return configuration
 }
 
-func getRuleForPath(pa v1beta1.HTTPIngressPath, i *v1beta1.Ingress) string {
+func getRuleForPath(pa v1beta1.HTTPIngressPath, i *v1beta1.Ingress) (string, error) {
 	if len(pa.Path) == 0 {
-		return ""
+		return "", nil
 	}
 
 	ruleType := i.Annotations[label.TraefikFrontendRuleType]
 	if ruleType == "" {
 		ruleType = ruleTypePathPrefix
 	}
 
+	pathReplaced := false
```

Comment:
Instead of just a boolean, how about using a string denoting the annotation that we found a replace rule in previously? That way, we can clearly signal to the user which other instance of the replace rule was violating.